### PR TITLE
Add License Scanning to the Pipeline

### PR DESCRIPTION
**Pull request by RemakingEden**
Implementation of the license scanner discussed here https://github.com/Project-MONAI/monai-deploy-workflow-manager/discussions/89

Signed-off-by: Joss Sparkes <joss.sparkes@gmail.com>

### CODE REVIEW

**Comment by RemakingEden** [comment]
So far all licenses have been dotnet, Apache 2.0 or MIT apart from one that is BSD. I have approved all of the former. I am unsure if the BSD license is acceptable. 

Moq, 4.17.2, https://raw.githubusercontent.com/moq/moq4/main/License.txt

After this is reviewed I can complete the pull request

**Comment by RemakingEden** [comment]
Still need to update the documentation for this

**Comment by RemakingEden** [comment]
Documentation is now added. I'm not 100% sure this correct and will need to be double checked that licenses mentioned are what we permit.

**Comment by Dusty-Meg** [comment]
Looks good to me, Just needs the confirmation about the Moq BSD license :+1:

**Review by Dusty-Meg — APPROVED** [review]

**Review by evberrypi — COMMENTED** [review]
LGTM

**Comment by dbericat** [comment]
LGTM. Approved

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -81,6 +92,38 @@ All source code files should start with this paragraph:
...
+We operate under the Apache 2.0 license. This means that we can only use packages that have certain permissive licenses. Below are the lists of licenses we can/maybe can allow.
+
+**Green list** (Can be committed without approval)
+
+- MIT
+- Apache 2.0
+
+**Amber list** (Can possibly be used but needs approval from the maintainers)
+
+- BSD
```
[the remark this answers is not in the document]
Comment (mocsharp) [reply CONTRIBUTING.md:106]: @dbericat approved [below](https://github.com/Project-MONAI/monai-deploy-workflow-manager/pull/90#issuecomment-1102998138)

**Review by mocsharp — APPROVED** [review]

**Pull request merged**